ClosedXML_Tests/Excel/Loading/LoadingWithSubtotalTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using ClosedXML.Excel;
+using NUnit.Framework;
+
+namespace ClosedXML_Tests.Excel
+{
+    [TestFixture]
+    public class LoadingWithSubtotalTests
+    {
+        [Test]
+        public void CanSuccessfullyCalcSubtotalFormulas()
+        {
+            using (var stream = TestHelper.GetStreamFromResource(TestHelper.GetResourcePath(@"TryToLoad\TableWithSubtotal.xlsx")))
+            {
+                var wb = new XLWorkbook(stream, new LoadOptions { EventTracking = XLEventTracking.Disabled, RecalculateAllFormulas = true });
+
+                Assert.AreEqual(6, wb.Worksheet(1).Cell(3, "B").Value);
+                Assert.AreEqual(24, wb.Worksheet(1).Cell(6, "B").Value);
+                Assert.AreEqual(62, wb.Worksheet(1).Cell(8, "B").Value);
+
+                Assert.AreEqual(24, wb.Worksheet(1).Cell(6, "D").Value);
+                Assert.AreEqual(68, wb.Worksheet(1).Cell(8, "D").Value);
+
+                Assert.AreEqual(3, wb.Worksheet(1).Cell(3, "F").Value);
+                Assert.AreEqual(24, wb.Worksheet(1).Cell(6, "F").Value);
+                Assert.AreEqual(62, wb.Worksheet(1).Cell(8, "F").Value);
+
+                Assert.AreEqual(6, wb.Worksheet(1).Cell(3, "H").Value);
+                Assert.AreEqual(24, wb.Worksheet(1).Cell(6, "H").Value);
+                Assert.AreEqual(254, wb.Worksheet(1).Cell(8, "H").Value);
+                Assert.AreEqual(192, wb.Worksheet(1).Cell(9, "H").Value);
+
+                Assert.AreEqual(6, wb.Worksheet(1).Cell(3, "J").Value);
+                Assert.AreEqual(24, wb.Worksheet(1).Cell(6, "J").Value);
+                Assert.AreEqual(254, wb.Worksheet(1).Cell(8, "J").Value);
+                Assert.AreEqual(192, wb.Worksheet(1).Cell(9, "J").Value);
+                Assert.AreEqual(510, wb.Worksheet(1).Cell(13, "J").Value);
+            }
```

Comment:
The test may be improved greatly if fill cells with formulas explicitly rather than open the file. In that case, it would be easier to see what cases are covered.

E.g.
```
var wb = new XLWorkbook();
var ws = wb.AddWorksheet();
ws.Cell("A1").Value = 1;
ws.Cell("B1").Value = 2;
ws.Cell("A2").Formula = "SUBTOTAL(9, A1:B1)";
ws.Cell("B2").Formula = "SUBTOTAL(9, A1:A2)";

Assert.AreEqual(3, ws.Cell("A2").Value);
Assert.AreEqual(1, ws.Cell("B2").Value);
// etc.

Reply:
Ok, the xlsx file has been deleted. Now the workbook, data and formulas are created directly in the test.